fitbenchmarking/results_output.py

```diff
@@ -78,8 +62,8 @@ def save_results_tables(software_options, results_per_test, group_name,
                                             weighted_str,
```

Comment:
Could you change this to be `group_name,x, weighted_str`?

Reply:
done